providers/stores/mongo.js

```diff
@@ -32,13 +32,15 @@ class MongoStore {
    * @param {EntityCoordinates} coordinates
    * @returns A list of matching coordinates i.e. [ 'npm/npmjs/-/JSONStream/1.3.3' ]
    */
-  async list(coordinates) {
+  async list(coordinates, expand = null) {
```

Comment:
feels odd to have a `list` that gets the contents. Sort of like having Bash `ls` to the equivalent of `cat`. feels more like a "query" operation. Perhaps `list` just gets renamed to `query"?

Reply:
Seems like this change is coming from your original commit.
Besides that, i agree with your comment. Do you want us to address it in the next commits?

Reply:
right. the original did just list the coordinates. With the addition of expand and returning content, the name should be updated. Need to see if the method is being used anywhere else, for anything else. Right now I'm personally leaning towards `find`